RULES.md

```diff
@@ -1,536 +1,1037 @@
 # Implemented rules
```

Comment:
I was thinking we can keep it here since this is the most used file (with the README.md)
Is it a best practice to have all .md files (except README.md) in a /docs folder?

Reply:
I think it could go either way. Normally I'd put all the .md files in the /docs folder (with the exception of README.md). But if you don't want to break existing links to RULES.md I think it's fine to leave it as well.